docs/openapiv2/apidocs.swagger.json

```diff
@@ -1707,8 +1707,7 @@
         "HIGHER_CONSISTENCY"
       ],
       "default": "UNSPECIFIED",
-      "description": "- UNSPECIFIED: Default if not set. Behavior will be the same as MINIMIZE_LATENCY\n - MINIMIZE_LATENCY: Minimize latency at the potential expense of lower consistency.\n - HIGHER_CONSISTENCY: Prefer higher consistency, at the potential expense of increased latency.",
-      "title": "Controls the consistency preferences when calling the query APIs.\nbuf:lint:ignore ENUM_ZERO_VALUE_SUFFIX\nbuf:lint:ignore ENUM_VALUE_PREFIX"
```

Comment:
It appears that the title is removed because there is now only one line of comment. Just adding a second comment line brings the title field back, using both lines 🤷